src/common.ts

```diff
@@ -125,8 +129,16 @@ export const packSDJWT: PackSDJWT = async (claims, disclosureFrame, hasher, opti
       }
     }
 
+    if (disclosureFrame?._decoyCount > 0) {
```

Comment:
@ragnika seems like current implementation support decoy digests only for object, doesn't it?
Specification specifies that both objects and arrays can have decoy digests.

Reply:
@linasi yes, complete forgot about the decoys in arrays.
Have just updated it to include that feature as well